### PR TITLE
chore: add scripts for fixing lint

### PR DESCRIPTION
**Pull request by snitin315**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [X] **metadata update**

### Motivation / Use-Case

Add scripts for fixing lint issues.

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,9 @@
     "lint:js": "eslint --cache .",
     "lint:types": "tsc --pretty --noEmit",
     "lint": "npm-run-all -l -p \"lint:**\"",
+    "fix:prettier": "npm run lint:prettier -- --write",
```
Comment (alexander-akait) [inline package.json:32]: I think `fmt` is better name, in other languages you use `fmt` or `format` command, for example:
- `rust` https://github.com/rust-lang/rustfmt
- `dart` https://dart.dev/tools/dart-format
- and etc, you can google it :smile:

Comment (snitin315) [reply package.json]: Yep, It's better 👍🏻

File: package.json [hunk 2]
```diff
@@ -29,6 +29,9 @@
     "lint:js": "eslint --cache .",
     "lint:types": "tsc --pretty --noEmit",
     "lint": "npm-run-all -l -p \"lint:**\"",
+    "fmt:prettier": "npm run lint:prettier -- --write",
+    "fmt:js": "npm run lint:js -- --fix",
```
Comment (alexander-akait) [inline package.json:33]: - `fix:js` fixes linter problems
- `fmt` (without `prettier`) format your code

File: package.json [hunk 3]
```diff
@@ -29,6 +29,9 @@
...
+    "fmt": "npm-run-all fix:js fix:prettier",
```
Comment (snitin315) [inline package.json:34]: Like this ?

```suggestion
    "fmt": "npm run lint:prettier -- --write",
    "fix:js": "npm run lint:js -- --fix",
    "fix": "npm-run-all fix:js fmt",
```

Comment (alexander-akait) [reply package.json:34]: Yes

**Review by alexander-akait — APPROVED** [review]

**Comment by snitin315** [comment]
what about this one

https://github.com/webpack/schema-utils/blob/2fc216fa72bfb2f96119920c93a25c25790aa289/package.json#L28

File: package.json [hunk 4]
```diff
@@ -25,10 +25,13 @@
     "build": "npm-run-all -p \"build:**\"",
     "commitlint": "commitlint --from=master",
     "security": "npm audit",
-    "lint:prettier": "prettier \"{**/*,*}.{js,json,md,yml,css,ts}\" --list-different",
+    "fmt:check": "prettier \"{**/*,*}.{js,json,md,yml,css,ts}\" --list-different",
     "lint:js": "eslint --cache .",
     "lint:types": "tsc --pretty --noEmit",
-    "lint": "npm-run-all -l -p \"lint:**\"",
+    "lint": "npm-run-all lint:js lint:types fmt:check",
+    "fmt": "npm run lint:prettier -- --write",
```
Comment (alexander-akait) [inline package.json:32]: `lint:prettier` -> `fmt:check`

Comment (snitin315) [reply package.json:32]: Oh yes, missed it.

```suggestion
    "fmt": "npm run fmt:check -- --write",
```

**Pull request merged**